src/Microsoft.AspNetCore.SignalR.Common/Microsoft.AspNetCore.SignalR.Common.csproj

```diff
@@ -10,11 +10,11 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="MsgPack.Cli" />
-    <PackageReference Include="System.Binary" />
-    <PackageReference Include="System.Buffers.Primitives" />
-    <PackageReference Include="System.Memory" />
-    <PackageReference Include="System.Numerics.Vectors" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
+    <PackageReference Include="System.Binary" Version="0.1.0-e171006-1" />
```

Comment:
Shouoldn't these version come from a central location? Are these versions correct (especially this one: 0.1.0-e171006-1)?